### ISSUE TITLE
InstructionAPI fails to decode some vector instructions on x86-64

### ISSUE THREAD

**Issue by mxz297** [dyninst/dyninst#573]
HPCToolkit is missing performance numbers for some code ranges. It turns out that InstructionAPI failed to decode some vector instructions. which lead to incomplete function ranges. 

Below are the unique opcodes we missed:

```
  6144f7:       8f e8 58 9e d0 30       vpmacsdd %xmm3,%xmm0,%xmm4,%xmm2
  78bb61:       c4 e3 e9 6a 09 00       vfmaddss (%rcx),%xmm0,%xmm2,%xmm1
  5b75b6:       c4 a3 71 6d 04 0e 00    vfmsubpd %xmm0,(%rsi,%r9,1),%xmm1,%xmm0
  786a2e:       c4 c3 c1 7b 44 c7 f8    vfnmaddsd -0x8(%r15,%rax,8),%xmm1,%xmm7,%xmm0
  788911:       c4 e3 e9 6f c9 00       vfmsubsd %xmm1,%xmm0,%xmm2,%xmm1
  788a4d:       c4 e3 51 6b 08 10       vfmaddsd %xmm1,(%rax),%xmm5,%xmm1
  667e23:       8f e8 48 ce ca 03       vpcomd $0x3,%xmm2,%xmm6,%xmm1
  7876b5:       8f e8 68 a2 c5 40       vpcmov %xmm4,%xmm5,%xmm2,%xmm0
  4e714d:       8f e9 70 9a cc          vpshad %xmm1,%xmm4,%xmm1
  473b76:       c4 e3 49 79 05 20 7e    vfnmaddpd %xmm1,0x337e20(%rip),%xmm6,%xmm0 
```

**Comment by mxz297** [comment]
Two more missing instructions are reported:

```
 4d832e:   8f e8 d0 a3 05 18 33    vpperm 0x2e3318(%rip),%xmm6,%xmm5,%xmm0        
 4d8335:   2e 00 60
 72cd47:       c4 e3 79 0b c0 0a       vroundsd $0xa,%xmm0,%xmm0,%xmm0
```

**Comment by mxz297** [comment]
The fix for this issue is in #575
